mutations/addLabel/action.yml

```diff
@@ -55,11 +55,11 @@ runs:
             }
           )
 
-          const node = repoLabels.nodes.find(({name} ) => name === '${{inputs.LABEL}}' )
+          let node = repoLabels.nodes.find(({name} ) => name === '${{inputs.LABEL}}' )
 
           if(!node) {
-            core.warning('Could not find label ${{inputs.LABEL}} in the target repo')
-            return
+            core.warning('Could not find label ${{inputs.LABEL}} in the target repo, creating')
+            node = await github.rest.issues.createLabel({$owner: String!, $repo: String!, name })
```

Comment:
- `$owner` is graphql syntax so that wont work here
- name will be undefined here ,on line 58 its destructured from the graphql query